network-api/networkapi/buyersguide/templatetags/yes_no.py

```diff
@@ -7,18 +8,22 @@
 def yes_no(value):
     """Converts boolean to yes or no string"""
     if value is None:
-        return "Unknown"
+        return gettext('Unknown')
```

Comment:
we can't do this here: these are not strings that end up human-readable anywhere, but are comparator values, and are used for deciding which labels to put on a page, using `{% if value == "Unknown" %} ... {% else if ... %} ... `

No localization should be required here (and will in fact break things quite hard =D)